ui/admin/app/templates/scopes/scope/host-catalogs/host-catalog/hosts/index.hbs

```diff
@@ -11,38 +11,41 @@
     </table.header>
     <table.body as |body|>
       {{#each @model as |host|}}
-        <body.row as |row|>
```

Comment:
Changes to this file are probably unnecessary, since by hiding non-static catalogs, users cannot navigate to its hosts.